Bottom: 8b0b927a8b2dbea7e233300c165e817f47e7a521
Top:    e3afcf1fb7fcf6e81bafa3d6b290913e98aeebcf
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-24 17:14:30 +0800

Refresh of vhost-device-iotlb-api

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 6961630..0f45e3d 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -974,7 +974,6 @@ static long vhost_net_set_backend(struct vhost_net *n, unsigned index, int fd)
 
 	/* Verify that ring has been setup correctly. */
 	if (!vhost_vq_access_ok(vq)) {
-		printk("vhost_vq_access_ok()!\n");
 		r = -EFAULT;
 		goto err_vq;
 	}
